src/libs/Account_Manager.py

```diff
@@ -10,16 +10,13 @@ def __init__(self, consumer_key, consumer_secret, access_key, access_secret):
         self.__accessSecret = access_secret
         self.__api = None
 
-# Twitter API has a limited rate : https://developer.twitter.com/en/docs/basics/rate-limiting
-# Added wait_on_rate_limit to true to automatically wait for rate limits tor eplenish
-# Added wait_on_rate_limit_notify to true since it to print a notification when Tweepy is waiting for rate limits to replenish
     def login(self, proxy=None):
         auth = tweepy.OAuthHandler(self.__consumerKey, self.__consumerSecret)
         auth.set_access_token(self.__accessKey, self.__accessSecret)
         if proxy is None:
-            self.__api = tweepy.API(auth, wait_on_rate_limit=True, wait_on_rate_limit_notify=True)
```

Comment:
You need to keep wait limit and notify as True otherwise, if it hits the rate limit you will kick out from the app . Please check the API doc